### DISCUSSION TITLE
Some noob questions

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#95] [Q&A]
Hi! 
First I love what you have created for the public. This windows tool is very cool. I do have some questions though. I am very new to privacy and security. There is a lot of FUD and I just want to make the best decisions. 

My first question is should I start with a clean install of the OS or if I buy a new surface machine I will be okay to run the script?

2nd: Do I only need to run the script? Or should I do the script, compliance checking and hardening? I’m a little confused about all that. 

3rd how often do I need to run the script for like updates from Microsoft? 

4th do you also use a surface machine and if so what do you recommend?

4.5: in the video for total protection you put a 2 instead of a 1 why is that?

5th the privacy community is concerned about the telemetry from Microsoft, do you think there is a need to worry about that?

Thank you so much for the work you have done!

**Comment by HotCakeX** [comment]
Hi, thank you!

1. You can apply the hardening measures at any time, no clean installation needed. If you clean install your OS, whether on a new device or your current one, make sure it's up to date first by installing updates and then use the module to harden it.


2. Compliance checking is just an extra feature, not necessary to be used. All you need to do is to install the module `Install-Module -Name Harden-Windows-Security-Module -Force` and then use the `protect-WindowsSecurity` cmdlet.

3. You can use the watch button on the repository and be notified using Email or GitHub notifications when there is a new version.

like this, just check the box for releases
![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/3629897d-3675-41ae-8397-2ae8fc82dcc4)

4. Yes, I use Surface device too, Surface book 3 I have to upgrade soon. Depending on your use cases and budget you need to decide which one to choose :)  https://www.microsoft.com/en-US/store/b/shop-all-microsoft-surface

5. I used option 2 for Microsoft security baseline + [optional overrides](https://github.com/HotCakeX/Harden-Windows-Security/wiki/Overrides-for-Microsoft-Security-Baseline)

6. There is absolutely nothing to worry about. Just minimize your exposure to 3rd party services because not all of them handle user data properly.

Feel free to ask any other related questions! Thanks, have a good one!

**Reply by ghost** [reply]
Thank you so much for getting back to me!  Do you recommend doing a clean install or if buying a new surface from Microsoft you wouldn't worry about it? Also do you recommend using a Microsoft account when setting up a new machine? 

Thanks!

**Reply by HotCakeX** [reply]
There is no need to do a clean install when you buy a new Surface device.
Yes, definitely use Microsoft account when setting up device, setup Windows Hello so you can authenticate with face, PIN etc. and use OneDrive for keeping your files safe and accessible.

**Reply by ghost** [reply]
One more question! Do you recommend buying a laptop already with windows 11 pro or one with windows 11 and buy the pro upgrade key? Thank you for all your help!

**Reply by HotCakeX** [reply]
Both options are okay, I'd just get a device with Windows 11 pro preloaded on it. When you sign into the device with your Microsoft account, the license will be tied to your account as digital license, with that you can [reactivate](https://support.microsoft.com/en-us/windows/activate-windows-c39005d4-95ee-b91e-b399-2820fda32227) your Windows whenever you wipe your device or clean install.

**Reply by ghost** [reply]
oh that's good to know! So is your script basically a run it and "forget about it" kind of thing? I will be protected from 99% of bad things? haha

**Reply by HotCakeX** [reply]
Yes, exactly ^^ all the features are already in the Windows, we just turn them on and fine-tune them 🙂

New features however are constantly added to the script so every couple of months you can run it once to catch up on them

**Reply by ghost** [reply]
Sorry for all the questions! Do you recommend using native Microsoft apps like their email client? Or download outlook? I have heard good things about thunderbird

**Reply by HotCakeX** [reply]
No it's okay,
I use the new Outlook app which replaces the built in Mail app, both of them are great, i'd recommend those. I don't use 3rd party software and solutions when 1st party alternatives already exist, this also minimizes the attack surface and prevents data leaks through 3rd parties.

**Reply by ghost** [reply]
Thank you for all your help!

**Reply by ghost** [reply]
This might sound really off topic but since you are very knowledgeable in cybersecurity do you have a mobile recommendation? I’m an Apple person myself.

**Reply by HotCakeX** [reply]
I use Samsung so I'd recommend that, but I don't really do important stuff on my phone.
